doc/design/cluster_train/data_dispatch.md

```diff
@@ -107,13 +109,14 @@ trainer.train(batch_reader, ...)
 
 使用下面命令，可以把本地的数据上传到存储集群中。
 
-```bash
-paddle cp filenames pfs://home/folder/
+```bash  
+paddle pfs cp filenames /pfs/folder/
```

Comment:
跟@wangkuiyi 讨论了一下，他的建议是：
```text
# config file
[datacenter_1]
username=wuyi
usercert=wuyi.pem
userkey=wuyi-key.pem
endpoint=datacenter1.paddlepaddle.org

[datacenter_2]
username=wuyi
usercert=wuyi.pem
userkey=wuyi-key.pem
endpoint=datacenter2.paddlepaddle.org
```
To upload:
```bash
paddle pfs cp local_file.txt /pfs/datacenter_2/home/david/ # do not have permission to /pfs/datacenter_2/home/john/
```
In `train.py`, one can reference the file by:
```text
/home/david
```
We need the home directory because in the future we may want john's directory be visible to david.